src/basis/basis.js

```diff
@@ -7,6 +7,12 @@ import { ATOMIC_COORD_UNITS, HASH_TOLERANCE } from "../constants";
 import { Lattice, nonPeriodicLatticeScalingFactor } from "../lattice/lattice";
 import math from "../math";
 
+
+/**
+ * Maximum number of atoms allowed in materials designer basis or a file upload basis.
```

Comment:
This message is against the encapsulation of concepts. A person using this library has to know about materials designer and file upload to understand this. A better approach would be to call this something independent of the other packages, like `maximumNumberAtomsSmallMolecule`. Then explain the logic that everything beyond that is to be considered a "large" molecule and thus subject to different processing.

Reply:
Alternatively, we can just include `nonPeriodicMaxAtomCount` inside the `Basis` class as a static variable

Reply:
changed to static variable.